components/org.wso2.carbon.identity.data.publisher.authentication.audit/src/main/java/org/wso2/carbon/identity/data/publisher/authentication/audit/AuthenticationAuditLoggerUtils.java

```diff
@@ -80,7 +81,11 @@ public static AuthenticationAuditData createAuthenticationAudiDataObject(Event e
             authenticationAuditData.setAuthenticatedIdps(getIdentityProviderList(context, status));
 
         }
-
+        if (StringUtils.isNotBlank(authenticationAuditData.getTenantDomain())) {
+            String userId = IdentityUtil.getInitiatorId(getUserNameForAuthenticationStep(params),
```

Comment:
check how AuthenticatedUser is resolved in https://github.com/wso2-extensions/identity-data-publisher-authentication/pull/104/files#diff-7e633e100b942bead2770c632a75a52a7f96ee94ae495a277f5c33e043998fa7R67-R82

Reply:
resolved

Reply:
resolved